### ISSUE TITLE
InjectorFactory combines Modules silently

### ISSUE THREAD

**Issue by panghy** [HubSpot/dropwizard-guicier#54]
In dropwizard-guice, InjectorFactory used to be:

```
/**
 * Factory to create Guice Injector with supplied Stage and List of Modules.
 *
 * Idea behind separating this out is to enable integrating applications to
 * use alternate Guice factories like, - Mycila
 * (https://code.google.com/p/mycila/), - Governator
 * (https://github.com/Netflix/governator)
 */
public interface InjectorFactory {
    Injector create(final Stage stage, final List<Module> modules);
}
```

Now, create() does not take a list of modules and in GuiceBundle it uses Modules.combine():

```
this.injector = injectorFactory.create(guiceStage, Modules.combine(modulesBuilder.build()));
```

This isn't as useful since governator can handle a list of modules (with more graceful handling of double-binding). Seems like a fairly simple change to me to re-add support to the interface.

**Comment by jhaber** [comment]
Thanks for the suggestion, let me know if #56 addresses your use-case. Unfortunately you'd need to implement both methods, but the library should only call the overload with multiple modules (you could make the single module version throw an UnsupportedOperationException I guess)

**Comment by panghy** [comment]
Sounds fine to me, thanks!

**Comment by jhaber** [comment]
Fixed in 1.3.5.2 which should show up in maven central soon

**Comment by panghy** [comment]
Thanks
